pkg/virt-handler/node-labeller/arch_labeller.go

```diff
@@ -41,6 +46,7 @@ type archLabeller interface {
 	hasHostSupportedFeatures() bool
 	supportsHostModel() bool
 	arch() string
+	prepareLabels(n *NodeLabeller, node *v1.Node, cpuModels []string, cpuFeatures cpuFeatures, hostCpuModel hostCPUModel, obsoleteCPUsx86 map[string]bool) map[string]string
```

Comment:
I guess that `	shouldLabelNodes() bool` and [this](https://github.com/kubevirt/kubevirt/blob/d0dd376266d025ff02e612a1cc733526904dc77d/cmd/virt-handler/virt-handler.go#L317) if statement are not needed anymore right?